spec/support/request_helpers.rb

```diff
@@ -89,23 +89,11 @@ def check_400_on_invalid_json
     end
   end
 
-  def check_content_type_header
```

Comment:
we attempted to explain the reasoning in [this commit message](https://github.com/alphagov/publishing-api/commit/4ff4e2486fe34351837e83cac9e7ca2268ebeb06)

Neither Daniel nor I could understand the reason for doing this given that we're either:

a) propagating a successful response from content store, which is always json (because we json parse it and a parse failure would be an error)
b) returning an error, in which case we construct a json response body and response code, rather than propagating the content stores response

we couldn't think of a scenario in which propagating the response header made sense, but we may have missed something?